tests/to_string.rs

```diff
@@ -0,0 +1,17 @@
+use edn_derive::Serialize as SerializeEdn;
+use edn_rs::Serialize;
+
+#[derive(SerializeEdn, Clone)]
+pub struct Person {
+    name: String,
+    age: usize,
+}
+
+fn main() {
+    let person = Person {
+        name: "joana".to_string(),
+        age: 290000,
+    };
+    assert_eq!(person.to_string(), "{ :name \"joana\", :age 290000, }");
```

Comment:
I think instead of calling `to_string` method directly on the struct, we should do `edn_rs::to_string(person)`. And have that funcion call `serialize` for us.

Implementing `Display`/`ToString` for the user is too intrusive, the user should do that however she/he wants. We should implement `serialize` and she/he should call our `edn_rs::to_string`.

What do you think?